lib/more_core_extensions/core_ext/math/slope.rb

```diff
@@ -0,0 +1,51 @@
+module MoreCoreExtensions
+  module MathSlope
+    module ClassMethods
+      # Finds the x coordinate given y, slope of the line and the y intercept
+      #
+      # x = (y - b) / m
+      # Where m is the slope of the line and b is the y intercept (the location where the line crosses the y axis)
+      def slope_x_intercept(y, slope, y_intercept)
```

Comment:
we may want keyword args here for caller readability, but that can be a follow up PR.